lib/Rex/Commands.pm

```diff
@@ -570,7 +570,9 @@ sub password {
 
 =head2 auth(for => $entity, %data)
 
-With this function you can modify/set special authentication parameters for tasks and groups. If you want to modify a task's or group's authentication you first have to create it.
+With this function you can modify/set special authentication parameters for tasks and groups.
+If you want to modify a task's or group's authentication you first have to create it.
+(Place the auth command after the task.)
```

Comment:
This limitation has been lifted on #596, fixing #402. I'm gonna remove this before merging.